src/main/java/com/google/codeu/servlets/MessageFeedServlet.java

```diff
@@ -0,0 +1,22 @@
+package com.google.codeu.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Handles fetching all messages for the public feed.
+ */
+@WebServlet("/feed")
+public class MessageFeedServlet extends HttpServlet {
+  
+ @Override
+ public void doGet(HttpServletRequest request, HttpServletResponse response)
+   throws IOException {
```

Comment:
This is a continuation of the previous line (aka a line wrap), so it should be indented twice, but there is only one indent here.

This is a style convention for line wrapping, which can vary from project to project and company to company, but generally, double-indenting a line wrap helps make it clearer.
https://google.github.io/styleguide/javaguide.html#s4.5-line-wrapping